### ISSUE TITLE
Simple content ingores restrictions on base type

### ISSUE THREAD

**Issue by Pretasoc** [mganss/XmlSchemaClassGenerator#237]
If have a schema with a construct like this:

``` xml
<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" elementFormDefault="qualified">
    <xs:simpleType name="SimpleTypeWithRestriction">
        <xs:restriction base="xs:normalizedString">
            <xs:minLength value="1"/>
            <xs:maxLength value="255"/>
        </xs:restriction>
    </xs:simpleType>

    <xs:complexType name="ComplexType">
        <xs:simpleContent>
            <xs:extension base="SimpleTypeWithRestriction"/>
        </xs:simpleContent>
    </xs:complexType>
</xs:schema>
```

This example produces the following output:

``` csharp
//------------------------------------------------------------------------------
// <auto-generated>
//     This code was generated by a tool.
//
//     Changes to this file may cause incorrect behavior and will be lost if
//     the code is regenerated.
// </auto-generated>
//------------------------------------------------------------------------------

// This code was generated by XmlSchemaClassGenerator version 2.0.0.0.
namespace Test
{
    using System.Collections;
    using System.Collections.Generic;
    using System.Collections.ObjectModel;
    using System.Xml.Serialization;
    
    
    [System.CodeDom.Compiler.GeneratedCodeAttribute("XmlSchemaClassGenerator", "2.0.0.0")]
    [System.SerializableAttribute()]
    [System.Xml.Serialization.XmlTypeAttribute("ComplexType", Namespace="")]
    [System.Diagnostics.DebuggerStepThroughAttribute()]
    [System.ComponentModel.DesignerCategoryAttribute("code")]
    public partial class ComplexType
    {
        
        /// <summary>
        /// <para xml:lang="de">Ruft den Text ab oder legt diesen fest.</para>
        /// <para xml:lang="en">Gets or sets the text value.</para>
        /// </summary>
        [System.Xml.Serialization.XmlTextAttribute()]
        public string Value { get; set; }
    }
}
```

As you can see, no `MinLengthAttribute` and `MaxLengthAttribute` are emitted for the `Value` property.

**Comment by mganss** [comment]
Thanks, fixed in 2.0.479.

**Comment by Pretasoc** [comment]
Wow, that was relly fast! Thank you!
